x-pack/filebeat/module/okta/system/config/pipeline.js

```diff
@@ -0,0 +1,162 @@
+            { from: "json.outcome.result", to: "okta.outcome.result" },
+            { from: "json.target", to: "okta.target" },
+            { from: "json.transaction.id", to: "okta.transaction.id" },
+            { from: "json.transaction.type", to: "okta.transaction.type" },
+            { from: "json.debugContext.debugData.deviceFingerprint", to: "okta.debug_context.debug_data.device_fingerprint" },
+            { from: "json.debugContext.debugData.requestId", to: "okta.debug_context.debug_data.request_id" },
+            { from: "json.debugContext.debugData.requestUri", to: "okta.debug_context.debug_data.request_uri" },
+            { from: "json.debugContext.debugData.threatSuspected", to: "okta.debug_context.debug_data.threat_suspected" },
+            { from: "json.debugContext.debugData.url", to: "okta.debug_context.debug_data.url" },
+            { from: "json.authenticationContext.authenticationProvider", to: "okta.authentication_context.authentication_provider" },
+            { from: "json.authenticationContext.authenticationStep", to: "okta.authentication_context.authentication_step" },
+            { from: "json.authenticationContext.credentialProvider", to: "okta.authentication_context.credential_provider" },
+            { from: "json.authenticationContext.credentialType", to: "okta.authentication_context.credential_type" },
+            { from: "json.authenticationContext.externalSessionId", to: "okta.authentication_context.external_session_id" },
+            { from: "json.authenticationContext.interface", to: "okta.authentication_context.authentication_provider" },
+            { from: "json.authenticationContext.issuer", to: "okta.authentication_context.issuer" },
+            { from: "json.securityContext.asNumber", to: "okta.security_context.as.number" },
+            { from: "json.securityContext.asOrg", to: "okta.security_context.as.organization.name" },
+            { from: "json.securityContext.domain", to: "okta.security_context.domain" },
+            { from: "json.securityContext.isProxy", to: "okta.security_context.is_proxy" },
+            { from: "json.securityContext.isp", to: "okta.security_context.isp" },
+        ],
+        mode: "rename",
+        ignore_missing: true,
+    });
+
+    var copyFields = new processor.Convert({
+        fields: [
+            { from: "okta.client.user_agent.raw_user_agent", to: "user_agent.original" },
+            { from: "okta.client.ip", to: "client.ip" },
+            { from: "okta.client.ip", to: "source.ip" },
+            { from: "okta.event_type", to: "event.action" },
+            { from: "okta.outcome.result", to: "event.outcome" },
+            { from: "okta.security_context.as", to: "client.as" },
+            { from: "okta.security_context.domain", to: "client.domain" },
+            { from: "okta.security_context.domain", to: "source.domain" },
+            { from: "okta.uuid", to: "event.id" },
```

Comment:
Can you also add the `add_id` processor to copy the `okta.uuid` to the events `_id` field. Until we have persistance available via the Filebeat input v7 refactoring, this is what will prevent duplicates if Filebeat is restarted and starts reading from the beginning.